packages/insomnia-app/app/ui/components/editors/body/graph-ql-editor.tsx

```diff
@@ -321,6 +323,54 @@ class GraphQLEditor extends PureComponent<Props, State> {
     }
   }
 
+  async _loadAndSetLocalSchema() {
+    const newState: Partial<State> = {
+      schemaFetchError: null,
+      schemaIsFetching: false,
+    };
+
+    const options: OpenDialogOptions = {
+      title: 'Import GraphQL introspection schema',
+      buttonLabel: 'Import',
+      properties: ['openFile'],
+      filters: [
+        // @ts-expect-error https://github.com/electron/electron/pull/29322
+        {
+          extensions: ['', 'json'],
+        },
+      ],
+    };
+
+    const { canceled, filePaths: paths } = await electron.remote.dialog.showOpenDialog(options);
```

Comment:
```suggestion
    const { canceled, filePaths } = await electron.remote.dialog.showOpenDialog(options);
```

I think we should not alias this.  It isn't being done to avoid a shadowing of `paths` somewhere else (so far as I can tell), and just adds cognitive overhead (to me).  Maybe it'd be good, in kind, to also do:

```ts
      const filePath = filePaths[0]; // showOpenDialog is single select
      const file = readFileSync(filePath);
 ```
 
 below as well